workflow-service-sdk/.gitignore

```diff
@@ -0,0 +1,21 @@
+*.class
```

Comment:
this file should be from the previous folder, no?

Reply:
Do you mean that there shouldn't be a `gitignore` file in this module? 
Or you don't like something inside the `.gitignore` file (ie. `*.class`)? 
This `.gitignore` file is auto generated by the openapi tool.
TBH, I've no idea on how to customize its generation. The only thing that I can do is to add the `.gitignore` file to `.openapi-generator-ignore` to prevent the generation and the manually "tune" the `.gitignore`.

Can you clarify a bit?

Reply:
ok, make sense.